spec/features/agent_searches_for_appointments_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.feature 'Agent searches for appointments' do
+  before do
```

Comment:
This makes more sense when expressed in something like `def and_appointments_exist` that you refer to in our already given-when-then style specs.